lib/pronto/formatter/text_formatter.rb

```diff
@@ -1,44 +1,53 @@
 module Pronto
   module Formatter
-    class TextFormatter
-      include Colorizable
+    class TextFormatter < Base
+      class TextMessageDecorator < SimpleDelegator
```

Comment:
@mmozuras moved `TextMessageDecorator` to `pronto/formatter/text_message_decorator.rb`